docs/source/publish_and_share_kedro_viz_on_aws.md

```diff
@@ -0,0 +1,119 @@
+# Publish and share on AWS
+
+This page describes how to publish Kedro-Viz on AWS to share it with others. It uses the spaceflights tutorial as an example.
+
+## Setup your kedro project 
+
+If you haven't installed Kedro {doc}`follow the documentation to get set up<kedro:get_started/install>`. 
+
+```{important}
+We recommend that you use the same version of Kedro that was most recently used to test this tutorial (0.19.1). To check the version installed, type `kedro -V` in your terminal window.
+```
+
+In your terminal window, navigate to the folder you want to store the project. Generate the spaceflights tutorial project with all the code in place by using the [Kedro starter for the spaceflights tutorial](https://github.com/kedro-org/kedro-starters/tree/main/spaceflights-pandas):
+
+
+```bash
+kedro new --starter=spaceflights-pandas
+```
+
+When prompted for a project name, you can enter anything, but we will assume `Spaceflights` throughout.
+
+When your project is ready, navigate to the root directory of the project. Install the dependencies from the project root directory by typing the following in your terminal:
+
+```bash
+pip install -r requirements.txt
+```
+
+Kedro-Viz requires specific minimum versions of `fsspec`, and `kedro` to publish your project. Ensure you have these correct versions by updating the `requirements.txt` file of the Kedro project to add the following:
+
+```text
+fsspec>=2023.9.0
+kedro>=0.18.2
```

Comment:
0.19.1?

Reply:
that's true but then we are implying shareable viz doesn't work with Kedro 18 which is not true and it does. 

i think the 0.19.1 from earlier was just that to get the best and latest be on the latest kedro version.